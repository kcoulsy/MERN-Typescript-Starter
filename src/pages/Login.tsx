import React from 'react';

import LoginForm from '../components/LoginForm/LoginForm';

const Login: React.FunctionComponent = (props) => {
    return (
        <div>
            <h3>Login Page</h3>
            <LoginForm {...props} />
        </div>
    );
};

export default Login;
