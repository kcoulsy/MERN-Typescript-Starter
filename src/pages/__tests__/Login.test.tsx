import React from 'react';
import { shallow } from 'enzyme';

import Login from '../Login';
import { shallowToJson } from 'enzyme-to-json';

describe('render Login component', () => {
    it('should render Login page component', () => {
        const wrapper = shallow(<Login />);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
