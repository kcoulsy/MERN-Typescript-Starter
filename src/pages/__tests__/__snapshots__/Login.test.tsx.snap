// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Login component should render Login page component 1`] = `
<div>
  <h3>
    Login Page
  </h3>
  <Connect(Component) />
</div>
`;
